modules/consolidated-scanner.adoc

```diff
@@ -8,4 +8,9 @@
 [role="_abstract"]
 {product-title-short} provides its own scanner, or you can configure an integration to use {product-title-short} with another vulnerability scanner.
 
-Beginning with version 4.4, Scanner V4, built on link:https://github.com/quay/claircore[ClairCore], provides scanning for language and operating system-specific image components. For version 4.4, {product-title-short} also uses the StackRox Scanner to provide some scanning functionality until that functionality is implemented in a future release.
+Beginning with version 4.4, Scanner V4, built on link:https://github.com/quay/claircore[ClairCore], provides scanning for language and operating system-specific image components. Currently, {product-title-short} also uses the StackRox Scanner to provide some scanning functionality, such as {op-system} node scanning, until that functionality is implemented in a future release.
```

Comment:
nit: "to provide other scanning functionality... To continue receiving full scanning support with the latest features, usage of both scanners is required". Something like that